src/IdentityServer/Validation/Models/ValidatedAuthorizeRequest.cs

```diff
@@ -259,6 +260,11 @@ public class ValidatedAuthorizeRequest : ValidatedRequest
     /// </summary>
     public string? PushedAuthorizationReferenceValue { get; set; }
 
+    /// <summary>
+    /// Is this a pushed authorization request?
+    /// </summary>
+    public bool IsPushedAuthorizationRequest { get => PushedAuthorizationReferenceValue.IsPresent(); }
```

Comment:
There is already an AuthorizeRequestType on the validated request that uses an enum to distinguish 3 states:
1. PAR
2. Authorize with Pushed Parameters
3. Authorize without Pushed Parameters

I don't know if it makes sense to include that distinction in any of these metrics - what do you think?

Regardless, my preference would be to use that existing property instead of introducing another flag. Earlier on in development of PAR, I only had a flag instead of the enum and confused myself.

Reply:
That's why I didn't find the property you mentioned in https://github.com/DuendeSoftware/IdentityServer/issues/1445#issuecomment-1775318325. I got a bit confused... I'll change to use the enum.